content/docs/hooks-intro.md

```diff
@@ -1,11 +1,11 @@
 ---
 id: hooks-intro
-title: Introducing Hooks
+title: Hooks പരിചയപ്പെടുത്തുന്നു
 permalink: docs/hooks-intro.html
 next: hooks-overview.html
 ---
 
-*Hooks* are a new addition in React 16.8. They let you use state and other React features without writing a class.
+React 16.8 ലെ പുതിയതായി ചേർക്കപ്പെട്ട ഫീച്ചർ ആണ് *Hooks*. ഇത് ഉപയോഗിച്ച് നിങ്ങൾക്കു state ഉം മറ്റു ഫീച്ചർ class ഇല്ലാതെ ഉപയോഗിക്കാൻ ആകും.
```

Comment:
class എഴുതാതെ സ്റ്റേറ്റും  മറ്റ് റീയാക്റ്റ് ഫീച്ചറുകളും ഉപയോഗിക്കാനാവും എന്നു പറയുന്നതല്ലേ കൂടുതൽ നല്ലത് ?

Reply:
yes, indeed. Added this correction.